sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/basicLogicalOperators.scala

```diff
@@ -774,13 +774,17 @@ case class Range(
       } else {
         (start + (numElements - 1) * step, start)
       }
+
+      val histogram = getHistogramStatistics
```

Comment:
Let's check `conf.histogramEnabled` here;
```
val histogram = if (conf.histogramEnabled) {
  Some(computeHistogramStatistics())
} else {
  None
}
```

Reply:
I feel its better to wrap a return value of `computeHistogramStatistics ` with `Some` in the caller side because the method always return a value.
https://github.com/apache/spark/pull/32498/files#diff-0ac2c89f8cc0d00e8fe7717b01697f36f20fe8abf2def09bcfde0ad84b30e467R814

Reply:
Done